tests/unit/test_gbq.py

```diff
@@ -650,6 +650,59 @@ def test_load_does_not_modify_schema_arg(mock_bigquery_client):
     assert original_schema == original_schema_cp
 
 
+def test_load_modifies_schema(mock_bigquery_client):
+    """Test of Issue # 670."""
```

Comment:
maybe just include a link to the issue directly as part of the the docstring?

Reply:
Thank you, just fixed it!